vtex/mod.ts

```diff
@@ -79,6 +79,11 @@ export interface Props {
    * @description Use VTEX as backend platform
    */
   platform: "vtex";
+
+  /**
+   * @description Rate limit for unique segment requests
```

Comment:
Explain better, I think user's don't call those "segments", it's an internal name